SPECS/python-tensorflow-io-gcs/python-tensorflow-io-gcs.spec

```diff
@@ -0,0 +1,50 @@
+%global pypi_name tensorflow-io-gcs
```

Comment:
What does gcs stand for?

Reply:
It stands for Google cloud storage, judging by the namespace defines in [their code](https://github.com/tensorflow/io/blob/master/tensorflow_io_gcs_filesystem/core/gcs_filesystem.cc#L36). @rikenm1 are we sure this package is necessary for our needs?